src/Tribe/Main.php

```diff
@@ -291,7 +291,7 @@ public function before_html_data_wrapper( $html ) {
 
 			$data_attributes = array(
 				'live_ajax'         => tribe_get_option( 'liveFiltersUpdate', true ) ? 1 : 0,
-				'datepicker_format' => tribe_get_option( 'datepickerFormat' ),
+				'datepickerFormat'  => tribe_get_option( 'datepickerFormat' ),
```

Comment:
The problem is that we cannot, since HTML might bug with `data-datepicker_format`, I changed it all the places we used it, on Core.